### PR TITLE
chore: aether 2.0.x

### PR DESCRIPTION
**Pull request by obdulia-losantos**

### CODE REVIEW

File: scripts/generate_env_vars.sh [hunk 1]
```diff
@@ -136,7 +136,7 @@ function gen_env_file {
 # ------------------------------------------------------------------
 # Releases
 # ==================================================================
-AETHER_VERSION=2.0.0
+AETHER_VERSION=2.0.2
```
Comment (obdulia-losantos) [inline scripts/generate_env_vars.sh:139]:
```suggestion
AETHER_VERSION=2.0.3
```

:thinking:

Comment (shawnsarwar) [reply scripts/generate_env_vars.sh:139]: We haven't released 2.0.3 yet have we?

Comment (obdulia-losantos) [reply scripts/generate_env_vars.sh:139]: _I see the future_ :crystal_ball:

Comment (obdulia-losantos) [reply scripts/generate_env_vars.sh:139]: Anyway, tests are not passing yet :disappointed_relieved:

**Review by shawnsarwar — APPROVED** [review]

**Pull request merged**